lib/routes/defaults.js

```diff
@@ -28,36 +28,44 @@ try {
 
 module.exports = (log, db) => {
 
-  function versionHandler(request, reply) {
+
+ async function versionHandler(request, h) {
     log.begin('Defaults.root', request)
 
-    function sendReply() {
-      reply(
-        {
-          version: version,
-          commit: commitHash,
-          source: sourceRepo
-        }
-      ).spaces(2).suffix('\n')
+     function getVersion() {
+         return new Promise(function (resolve, reject) {
+             // ignore errors and default to 'unknown' if not found
+             var gitDir = path.resolve(__dirname, '..', '..', '.git')
```

Comment:
And the indentation has gone up to 4 spaces per indent here, it should be 2 right?